deno.jsonc

```diff
@@ -1,6 +1,10 @@
 {
   "compilerOptions": {
     "checkJs": false,
+    "jsx": "react",
+    "jsxFactory": "h",
+    "jsxFragmentFactory": "Fragment",
+    "lib": ["deno.ns", "dom"],
```

Comment:
I don't think we should introduce a dependency on preact. The use is so minimal it's not worthwhile. Once we do get to making a more fancy server, it'll likely be out of the capi repo anyway.